PJ05/src/MessageSystemTest.java

```diff
@@ -0,0 +1,136 @@
+import Field.*;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Date;
+import java.util.UUID;
+
+import static org.junit.Assert.*;
+
+public class MessageSystemTest {
+    private final Database<User> db1 = (new Database<User>("userFileTest", User.class));
+    private final Database<Message> db2 = (new Database<Message>("messageFileTest", Message.class));
+    private final Database<Conversation> db3 = (new Database<Conversation>("conversationFileTest", Conversation.class));
```

Comment:
The databases are created by the MessageSystem, so you don't need to create them by yourself.